tools/openshift-ci/Dockerfile

```diff
@@ -37,6 +43,13 @@ RUN set -x && \
     yum clean all -y && \
     rm -rf /var/cache/yum /var/tmp/* /tmp/*
 
+#install ansible 
+RUN set -x && \
+    yum install -y epel-next-release  && \
+    yum install -y python3-pip  && \
+    pip3 install --upgrade pip && \ 
+    pip3 install ansible 
```

Comment:
We might need to install `openshift-ansible` instead of `ansible`. This is how it's installed in `ansible` image https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile

Need @gpei expertise on how qe should install openshift-ansible in our verification-tests image. We need the tool on RHEL node upgrade. Thank you!

Reply:
yeah, if it's for RHEL node upgrade, then we need `openshift-ansible` package to provide the related playbooks, and when installing this package via **yum**, `ansible-core` and `openshift-clients` would also be installed as the dependencies.  

```
.qa.[root@vm-10-0-77-151 yum.repos.d]# yum install openshift-ansible
Mirror RH Enterprise Linux 8 Appstream                                      631 kB/s | 6.7 MB     00:10    
Mirror RH Enterprise Linux 8 - BaseOS                                       228 kB/s | 2.4 MB     00:10    
ocp-414                                                                     310 kB/s | 3.2 MB     00:10    
Dependencies resolved.
============================================================================================================
 Package           Arch   Version                                             Repository               Size
============================================================================================================
Installing:
 openshift-ansible noarch 4.14.0-202303141716.p0.g5733377.assembly.stream.el8 ocp-4                    60 k
Installing dependencies:
 ansible-core      x86_64 2.13.3-2.el8_7                                      ci-rhel8-appstream-rpms 2.8 M
 openshift-clients x86_64 4.14.0-202303091954.p0.g07ee29c.assembly.stream.el8 ocp-4                    47 M

Transaction Summary
============================================================================================================
Install  3 Packages

Total download size: 50 M
Installed size: 148 M
Is this ok [y/N]: y
Downloading Packages:
(1/3): openshift-ansible-4.14.0-202303141716.p0.g5733377.assembly.stream.el  11 kB/s |  60 kB     00:05    
(2/3): openshift-clients-4.14.0-202303091954.p0.g07ee29c.assembly.stream.el 8.0 MB/s |  47 MB     00:05    
(3/3): ansible-core-2.13.3-2.el8_7.x86_64.rpm                               484 kB/s | 2.8 MB     00:05    
------------------------------------------------------------------------------------------------------------
Total                                                                       8.5 MB/s |  50 MB     00:05     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                    1/1 
  Installing       : openshift-clients-4.14.0-202303091954.p0.g07ee29c.assembly.stream.el8.x86_64       1/3 
  Installing       : ansible-core-2.13.3-2.el8_7.x86_64                                                 2/3 
  Installing       : openshift-ansible-4.14.0-202303141716.p0.g5733377.assembly.stream.el8.noarch       3/3 
  Running scriptlet: openshift-ansible-4.14.0-202303141716.p0.g5733377.assembly.stream.el8.noarch       3/3 
  Verifying        : ansible-core-2.13.3-2.el8_7.x86_64                                                 1/3 
  Verifying        : openshift-ansible-4.14.0-202303141716.p0.g5733377.assembly.stream.el8.noarch       2/3 
  Verifying        : openshift-clients-4.14.0-202303091954.p0.g07ee29c.assembly.stream.el8.x86_64       3/3 

Installed:
  ansible-core-2.13.3-2.el8_7.x86_64                                                                        
  openshift-ansible-4.14.0-202303141716.p0.g5733377.assembly.stream.el8.noarch                              
  openshift-clients-4.14.0-202303091954.p0.g07ee29c.assembly.stream.el8.x86_64                              

Complete!

```

Reply:
Do we have the yum repository for openshift-ansible? Note the baseurl needs to be publicly accessible

Reply:
@jhou1 working on adding repo for openshift-ansible with help of @gpei

Reply:
I tested on a clean CentOS-8 machine, for `ansible` package that satisfied our upgrade playbook could be `ansible-core < 2.14`, all the dependency rpm packages come from the CentOS-8 Appstream Repo, so no epel required, just 
`yum install -y 'ansible-core < 2.14'` is enough. 
As for `openshift-ansible`, as there's no public repo for it, especially for the version not GA yet, I suggest to use the playbooks from git repo https://github.com/openshift/openshift-ansible/ directly, it has the same content as openshift-ansible rpms, we can git clone it to the image, so that when upgrading the RHEL workers, we can use command:
`ansible-playbook -i <inventory> <path_to_the_repo>/openshift-ansible/playbooks/upgrade.yml  `

Reply:
Hi @gpei Thanks for your help! I'm seeing it's installed in ansible image by 
```
RUN yum install -y epel-release && \
    yum config-manager --enable built > /dev/null && \
    yum install --setopt=tsflags=nodocs -y \
      'ansible-core < 2.14' \
      python38-dateutil python38-urllib3 \
      openshift-ansible-test && \
```
Can we re-use the way in our image?
https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile#L25-L30

Reply:
@shellyyang1989 I also noticed this file this afternoon, but I didn't figure out how the `openshift-ansible-test ` package was installed in the end.

Reply:
@shellyyang1989 @gpei looks openshift-ansible is locally built using https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile#L26.  openshift-ansible-test can be installed by enabling built repository